frontend/shared/UserOrgSelect.test.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { spy } from 'sinon';
+
+import UserOrgSelect from './UserOrgSelect';
+
+const dateValue = new Date().toString();
+const id = 'hello-select';
+const label = 'Hello World';
+const name = 'select-name';
+const org1 = 'org-1';
+const org2 = 'org-2';
+const value = '';
+const initialProps = {
+  className: '',
+  id,
+  label,
+  touched: false,
+  error: null,
+  mustChooseOption: false,
+  name,
+  onChange: spy(),
+  orgData: [
```

Comment:
Even though this one is pretty small, I'd put this in another file so we have some organization fixture data to work with